app/migrations/versions/90b392e41d9f_.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
```

Comment:
can these be removed?

Reply:
I don't think so. It's a merge of two separate database migration heads (i.e. created when running `python manage.py db merge heads`)